org.jacoco.build/pom.xml

```diff
@@ -394,6 +405,11 @@
           <artifactId>xml-maven-plugin</artifactId>
           <version>1.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>exec-maven-plugin</artifactId>
+          <version>1.6.0</version>
```

Comment:
@Godin Ok, thanks for the Info! We can execute the documentation as part of the test. Or is that too much of a hack?